EVENT.md

```diff
@@ -1132,7 +1154,7 @@ effectively limits the number of unique sites covered per {source site, reportin
 To further reduce the possibility of a history reconstruction attack, the browser can also limit the number of `destination` sites registered per {source-site, 1 minute}.
 
 
-Additionally, to prevent one origin from using up the budget in the limit above, the browser can also limit the number of `destination` sites per {source site, reporting site, 1 minute}.
+Additionally, to prevent one origin from using up the budget in the limit above, the browser can also limit the number of `destination` sites per {source site, reporting site, 1 day}.
```

Comment:
According to the PR description, this change is being made for better privacy with FIFO. Is there a reason we should keep it as is for LIFO?

Reply:
The change is necessary for FIFO, and to avoid complexity, we will change this in either algorithm.